doc/sphinx/addendum/omega.rst

```diff
@@ -118,7 +120,7 @@ loaded by
 
 .. example::
 
-  .. coqtop:: all
+  .. coqtop:: all warn
```

Comment:
What about turning the warning off in the examples?

Reply:
I think it’s fine to show the warning in the manual.